build.sbt

```diff
@@ -478,14 +491,14 @@ lazy val `scio-sql`: Project = project
   .settings(
     description := "Scio - SQL extension",
     libraryDependencies ++= Seq(
-      "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
+      //"org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatVersion,
```

Comment:
Why is this commented out?

Reply:
If my understanding is correct, `scala-collection-compat` is needed when we compile in 2.12 to use features from the 2.13 standard library (not only collections) in a 2.12 codebase.

Naturally, this dependency is compiled for 2.13 and 3.0.0-RC1 and could be used without `cross`.
However `scio-core` depends on `algebird-core` with `.cross(CrossVersion.for3Use2_13)` which depends on `scala-collection-compat_2.13` which results in conflicts.

I believe that the next step for this is to add a `if (is scala 2.12) { depend on it }`. But I prefer to do it when I encounter the missing lib rather than now for all modules.

Reply:
Good, let's leave this as is for now. Opened #8 to track this.